clayui.com/plugins/gatsby-plugin-clay-css-tasks/gatsby-node.js

```diff
@@ -185,13 +186,88 @@ const generateCSSFiles = (pluginOptions) => {
 	);
 };
 
+const generateSassDocs = (pluginOptions) => {
+	const pjson = require('../../package.json');
+
+	var config = {
+		dest: path.join(pluginOptions.clayuiStatic, 'scss-api'),
+		// theme: path.join(__dirname, 'sassdoc-theme-clay-css'),
+		groups: {
+			alerts: 'Alerts',
+			applicationBar: 'Application Bar',
+			aspectRatio: 'Aspect Ratio',
+			badges: 'Badges',
+			breadcrumbs: 'Breadcrumbs',
+			breakpoints: 'Breakpoints',
+			bs4overwrites: 'Bootstrap 4 Overwrites',
+			buttons: 'Buttons',
+			cards: 'Cards',
+			caret: 'Caret',
+			clearfix: 'Clearfix',
+			customForms: 'Custom Forms',
+			drilldown: 'Drilldown',
+			dropdowns: 'Dropdowns',
+			float: 'Float',
+			forms: 'Forms',
+			globals: 'Globals',
+			gradients: 'Gradients',
+			grid: 'Grid',
+			hover: 'Hover',
+			image: 'Image',
+			inputGroups: 'Input Groups',
+			labels: 'Labels',
+			lineClamp: 'Line Clamp',
+			links: 'Links',
+			listGroup: 'List Group',
+			loaders: 'Loaders',
+			managementBar: 'Management Bar',
+			menubar: 'Menubar',
+			modals: 'Modals',
+			multiStepNav: 'Multi Step Nav',
+			nav: 'Nav',
+			navbar: 'Navbar',
+			navigationBar: 'Navigation Bar',
+			pagination: 'Pagination',
+			panels: 'Panels',
+			popovers: 'Popovers',
+			progressBars: 'Progress Bars',
+			quickAction: 'Quick Action',
+			sheet: 'Sheet',
+			sideNavigation: 'Side Navigation',
+			sidebar: 'Sidebar',
+			slideout: 'Slideout',
+			stickers: 'Stickers',
+			tables: 'Tables',
+			tbar: 'Tbar',
+			timelines: 'Timelines',
+			toggleSwitch: 'Toggle Switch',
+			tooltip: 'Tooltip',
+			type: 'Type',
+			undefined: 'Clay CSS',
+			utilities: 'Utilities',
+			vendorPrefixes: 'Vendor Prefixes',
+		},
+		package: {
+			description: pjson.description,
+			homepage: path.join('..', 'docs', 'get-started', 'index.html'),
+			license: pjson.license,
+			title: 'Clay CSS',
+			version: pjson.version,
+		},
+	};
+
+	sassdoc(path.join(pluginOptions.clayCssSrc, 'scss'), config);
```

Comment:
I had a different idea of this, actually I didn't want to use `sassdoc` to generate the static files but if it was possible to use the API to return the JSON data and so we could use it in different places in the documentation, for example, listed in the tab of `HTML/CSS` of each component. But we can explore that later.

Reply:
I didn't know they had an API we could use. I have no idea how to use it, but it sounds good.

Reply:
Yeah, this seems to be possible just by calling `sassdoc.parse` which will generate the raw data in a JSON.